service/version_bundles.go

```diff
@@ -3,13 +3,13 @@ package service
 import (
```

Comment:
Can we get rid of this file completely? and replace that with `[]versionbundle.Bundle{project.VersionBundle(provider)}`?

Reply:
Done